server/utils/referralUtils.ts

```diff
@@ -68,7 +68,10 @@ export default class ReferralUtils {
         heading: 'Referral information',
         items: [
           {
-            statusTag: ReferralUtils.taskListStatusTag('not started'),
+            statusTag: ReferralUtils.taskListStatusTag(
+              referral.hasReviewedProgrammeHistory ? 'completed' : 'not started',
+              'accredited-programme-history-tag',
```

Comment:
`programme-history-tag`? Don't think we've used "accredited" in the history context before